src/components/BulkUploaderFilePicker.tsx

```diff
@@ -105,10 +111,8 @@ export const BulkUploaderFilePicker = () => {
   };
 
   /**
-   * Provided a list of files
-   * from either a native file picker
-   * or the Drag and Drop API,
-   * validates and uploads the files.
+   * Provided a list of files from either a native file picker
```

Comment:
Whoops I didn't realize this changed the original file (but like, also, these are not README lines and so the concept of breaking so frequently is actually a bad UX!  I imagine prettier will update these in future so I'm happy to undo them for this PR)

Reply:
Don't worry about it.